### ISSUE TITLE
Question on Normal Distribution with large target

### ISSUE THREAD

**Issue by matthiasschuurmans** [stanfordmlgroup/ngboost#162]
Hi, thanks a lot for this package! I'm a great fan of boosters, and this package was just what I needed to enable some next steps in projects I'm working on.

I've been playing around with it on my data, and found that the Normal Distribution becomes a poor fit on datasets where the target is large, and am hoping to better understand why this happens. My target varies between 0 and ~50k. Some experiments that I did (keeping DecisionTreeRegressor as the base):

- Divide target by 1000, fit becomes good. I could of course just go with this, but like I said, I want to understand what is happening
- Use Exponential Distribution instead, fit is good
- Allow for ridiculous overfitting with hyperparameters (max_depth 1k, n_estimators 10k, min_sample_split 2 etc), fit remains bad (though just a tiny bit better than with less overfitting hyperparameter settings)

To make sure it's not my dataset, I also did a test with (a simplified version of) the example in the readme of this repo with the Boston housing dataset, where I multiplied the target by 1000, just to see how well it would fit (no split in train test):

```
from ngboost import NGBRegressor
from sklearn.datasets import load_boston
import plotly.graph_objects as go
from plotly.offline import plot

X, Y = load_boston(True)
Y *= 1000

ngb = NGBRegressor()
ngb = ngb.fit(X, Y)
preds = ngb.predict(X)

fig = go.Figure()

trace = go.Scatter(y=Y, name='Y', line=dict(color='orange'))
fig.add_trace(trace)

trace = go.Scatter(y=preds, name='Pred', line=dict(color='blue'))
fig.add_trace(trace)

plot(fig)
```

See the picture for the result:

![boston_multiply](https://user-images.githubusercontent.com/9294778/89174573-5ebd0000-d586-11ea-86a3-af901689562b.PNG)

Versus a good fit if the target is not multiplied by 1000:

![boston_no_multiply](https://user-images.githubusercontent.com/9294778/89174582-62e91d80-d586-11ea-8bf2-1be6b3158f07.PNG)

Can you help me understand this behavior? I've gone through both the paper and the code several times, I can identify some differences between the good and bad fit experiments, but I'm not comfortable enough (yet) with this material to have an educated guess on what is going on. Thank you in advance!

**Comment by avati** [comment]
Hi @matthiasschuurmans , this sounds strange, and I might have heard about a similar issue from another user. Could you please share the data/code that results in the poor fit and create the plot? I can have a look at it.

**Comment by matthiasschuurmans** [comment]
Hi @avati , thanks for looking into it. I placed the code in the issue, but just to be sure here it is as well. Please note the `Y *= 1000` that results in the bad fit, if you comment it out you get the second plot.

```
from ngboost import NGBRegressor
from sklearn.datasets import load_boston
import plotly.graph_objects as go
from plotly.offline import plot

X, Y = load_boston(True)
Y *= 1000

ngb = NGBRegressor()
ngb = ngb.fit(X, Y)
preds = ngb.predict(X)

fig = go.Figure()

trace = go.Scatter(y=Y, name='Y', line=dict(color='orange'))
fig.add_trace(trace)

trace = go.Scatter(y=preds, name='Pred', line=dict(color='blue'))
fig.add_trace(trace)

plot(fig)
```

**Comment by alejandroschuler** [comment]
(for posterity: similar issues noted in #154 and #135)

**Comment by avati** [comment]
thanks @matthiasschuurmans , i see your code now in the original post as well (somehow it didn't make it to the email notification). I'll have a look and post any updates.

**Comment by ryan-wolbeck** [comment]
@matthiasschuurmans this should be fixed if you adjust your version number to 0.2.2

**Comment by matthiasschuurmans** [comment]
Thanks a lot folks, indeed it is fixed after upgrading to 0.2.3 from the PR.
